src/contrib/persistence/Akka.Persistence.Query.InMemory.Tests/Akka.Persistence.Query.InMemory.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <Import Project="..\..\..\common.props" />
+    <Import Project="..\..\..\xunitSettings.props" />
+
+    <PropertyGroup>
```

Comment:
Needs to be marked as `<IsPackable>false</IsPackable>` - otherwise this will get shipped as a NuGet package.

Reply:
I'll fix that real fast

Reply:
@Arkatufus I took care of it